runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/ProcessInstance/ProcessInstance.thunks.ts

```diff
@@ -48,6 +48,18 @@ export const getProcessInstance = createAsyncThunk<IProcessInstance, {
     },
 );
 
+export const getSubProcesses = createAsyncThunk<IProcessInstance[], {
+    processInstanceId: string;
+}>(
+    'processInstances/getSubProcesses',
+    ({ processInstanceId }) => 
+        Axios.get<IProcessInstance[]>(`/api/process-instances/${processInstanceId}/subprocesses`)
+            .then((response) => {
+                if(response.data.length > 0) return response.data;
+                throw new Error('Could not found any sub processes');
```

Comment:
imo this logic should be with your dispatch in the component. We need to provide a translation here + If someone were to use this thunk, he/she might be surprised that the request was completed with a 200 status code but got an error

Reply:
moved logic to the dispatch.
there's translation

> what's missing in getSubProcessesResponse
yeah that proves the point. throw error removed

done